scripts/setup-test-data/src/setup_test_data/cli.py

```
@@ -0,0 +1,410 @@
+                "body": r.body,
+                "assets": assets_out,
+                "created_at": r.published_at,
+            }
+        )
+
+    with open(filepath, "w") as f:
+        json.dump(data, f, indent=2)
+    print(f"  Saved releases metadata: {filepath}")
+
+
+def main(args: Optional[list[str]] = None) -> None:
```

Ruff (0.15.15)

[warning] Too many branches (18 > 12)

(PLR0912)